src/RageThreads.cpp

```diff
@@ -682,11 +682,16 @@ void LockMutex::Unlock()
 
 	mutex.Unlock();
 
-	if( file && locked_at != -1 )
+	constexpr std::uint_fast64_t INVALID_TIME = static_cast<std::uint_fast64_t>(-1);
```

Comment:
You can drop std:: prefixes on types imported from cstdint (which we should be including in the files that use them btw). Here and everywhere in this PR.
https://google.github.io/styleguide/cppguide.html#Integer_Types